src/TypeScript.UnitTests/EslintBridgeClient/EslintBridgeProcessTests.cs

```diff
@@ -354,5 +355,14 @@ private static void SafeKillProcess(Process process)
                 // do nothing
             }
         }
+
+        private static string GetDefaultParameters()
```

Comment:
Having a unit test that just contains a cut-and-paste from the product code seems a bit weak. I'm not sure how we'd improve it, though.